lib/tasks/import_csvs.rake

```diff
@@ -49,20 +36,38 @@ task :import => [:environment] do
       }
     )
   end
+  puts "#{Item.count} Items created"
 
-  puts 'Importing merchants...'
-  CSV.foreach('./db/data/merchants.csv', headers: true, header_converters: :symbol, convert: :all) do |row|
-    Merchant.create(row.to_h)
+  puts 'Importing invoices...'
+  CSV.foreach('./db/data/invoices.csv', headers: true, header_converters: :symbol) do |row|
+    Invoice.create!(row.to_h)
   end
+  puts "#{Invoice.count} Invoices created"
 
-  puts 'Importing transactions...'
-  CSV.foreach('./db/data/transactions.csv', headers: true, header_converters: :symbol, convert: :all) do |row|
-    Transaction.create(row.to_h)
+  puts 'Importing invoice items...'
+  CSV.foreach('./db/data/invoice_items.csv', headers: true, header_converters: :symbol) do |row|
+    InvoiceItem.create!({
+      id: row[:id],
+      item_id: row[:item_id],
+      invoice_id: row[:invoice_id],
+      quantity: row[:quantity],
+      unit_price: row[:unit_price].to_f/100,
+      created_at: row[:created_at],
+      updated_at: row[:updated_at]
+      }
+    )
   end
+  puts "#{InvoiceItem.count} InvoiceItems created"
 
-  ActiveRecord::Base.connection.tables.each do |t|
-    ActiveRecord::Base.connection.reset_pk_sequence!(t)
+  puts 'Importing transactions...'
+  CSV.foreach('./db/data/transactions.csv', headers: true, header_converters: :symbol) do |row|
+    Transaction.create!(row.to_h)
   end
+  puts "#{Transaction.count} Transactions created"
 
-  puts "CSV successfully imported into database"
+  if Customer.count == 1000 && Merchant.count == 100 && Item.count == 2483 && Invoice.count == 4843 && InvoiceItem == 21687 && Transaction.count == 5595
```

Comment:
This statement is not necessary, but, will help determine if all databases have been seeded.